src/windows/scenery.c

```diff
@@ -33,7 +33,7 @@
 #include "../world/sprite.h"
 #include "dropdown.h"
 #include "../interface/themes.h"
-
+#include "../network/network.h"
```

Comment:
Same here, an empty line between the `include`s and `define`s helps with readability